README.md

```diff
@@ -1,20 +1,43 @@
 # \*IPs - Star Improvement Proposals
 *by Zachary Belford, Shane Jonas, Donald McIntyre, Yaz Khoury, Anthony Lusardi, Tomaz Kariz*
 
-Star Improvement Proposals are an open standard for contributors to propose changes, additions, or deprecations of core protocol specifications.
+Star Improvement Proposals is an extension of the BIP/EIP/ECIP process, with some changes to allow for decentralization of the process. Its fundamental change involves determinism of proposal numbers and content-based versioning. 
 
 The Star is meant to be replaced with the prefix/accronym of the users choice, though the prefix MUST be the same for all participants of the particular protocol specification.
 
 For example, All forks of the Star Improvement Proposal relating to the bitcoin network MUST be called `BIP`. The differentiator being the location/domain/github organization to which the \*IP belongs.
 
-Star Improvement Proposals are versioned by the content of their name. That is, in the case of making a proposal, the `version` identifier of the proposal MUST be derived by the last 8 characters of the SHA-3-512 of the proposal name. This `version` naming scheme is deterministic in that groups or individuals may choose to include another's proposal without any numbering conflicts.
+# Changes to BIP/EIP/ECIP
+## Version
+Star Improvement Proposals are versioned by the content of their name. First we create a hash of the preamble, then we use a substring of it to create the version hash.
 
-Once you have your \*ip repo up, people submit proposals to it via pull requests, adding a file named {networkName}IP-{proposalName}-{version}. Once the proposal is implemented by your team / group, you may signal your readyness by merging the pull request.
+### Preable hash
+SHA-3-512 of the proposal [preamble](https://github.com/ethereumclassic/ECIPs/blob/master/ECIP-1.sample.md#what-is-an-ecip).
```

Comment:
And as raised over at https://github.com/BelfordZ/starIPs/issues/7, I think SHA3 512 is overkill and less accessible than other comparable functions.

Reply:
@whilei The intent was to reduce scope of change in this PR. Should make another PR to change this to one that is preferred.

Reply:
Resolving this as Anthony's Base58 PR is merged here as an extra layer on top of SHA3-512.